### PR TITLE
Improve UI for attendance on PersonCard

### LINKED ISSUE #91: Improve UI for Attendance on Person Card

### PR DESCRIPTION
**Pull request by Anant1902**
Improved UI for each Person Card, which now shows Attendance more clearly. Resolves #91.

Patch Coverage is 0% since there was no pre-existing test file for PersonCard.java.

Screenshot 2024-03-27 at 4 37 50 PM

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css
Comment (saiutkarsh33) [inline src/main/resources/view/DarkTheme.css:1]: Great use of javafx here!

**Review by saiutkarsh33 — APPROVED** [review]
LGTM! First of our UI Changes!

**Review by ForAeons — APPROVED** [review]
Truly the frontend god. Fantastic work

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -32,7 +32,15 @@
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-      <FlowPane fx:id="attendance" styleClass="container" />
+      <HBox spacing="5" alignment="CENTER_LEFT">
+        <Label fx:id="att_id" styleClass="cell_medium_label" text="Week's Attended: ">
```
Comment (tituschewxj) [inline src/main/resources/view/PersonListCard.fxml:36]: A more correct phrasing would be `Weeks Attended`.
```suggestion
        <Label fx:id="att_id" styleClass="cell_medium_label" text="Weeks Attended: ">
```

**Review by tituschewxj — COMMENTED** [review]
Thanks for improving this!

**Pull request merged**